docs/reference/gettingstartedexample/gse-acq-help.md

```diff
@@ -111,37 +119,59 @@ Refer to this document to understand the elements of the getting started example
 - **Serial Channel** - Selects the serial channel number for the image data to be displayed if more than two serial input channels are active.
     > This control may be modified while the VI is running.
-
 - **Source Rate (fps)** - Displays the frame rate in frames per second at which the image data is being received.
 - **Update Rate (fps)** - Displays the rate in frames per second at which the image display indicator is being updated.
     > Note: If multiple frames have arrived since the last update, only the most recent frame is displayed and the others are skipped in an attempt to keep up with the acquisition.
+
 ### Serial Channel Packets Tab (First or Second)
 > The getting started example displays packet data for the first two channels specified in the Channel Configurations array at the completion of the acquisition. If other packet data is desired, either re-order the active channels in the configuration array on the Serial Channel tab or use the TDMS File Viewer utility.
+
+> Note: If displaying a large number of packets, the VI may appear unresponsive for a period of time after the acquisition completes while the packet data is processed.
+
 - **Acquired Packets** - Displays the LLP packet information for the number of packets specified in the **Logged Packets to Display** control starting at the first logged packet.
-    > Note: If displaying a large number of packets, the VI may appear unresponsive for a period of time after the acquisition completes while the packet data is processed.
+  #### Table of Descriptions for Acquired Packets Columns
+
+  | Column | Description |
+  | ------ | ----------- |
+  | Index | The order packets were received. |
+  | Timestamp (s) | The time in seconds when the packet was received. Timestamps are relative to a time immediately after the FPGA bitfile is downloaded and run, not the start of the acquisition. This allows capturing I2C and GPIO timestamps during configuration before the acquisition starts. |
+  | Source | The unique timestamp identifier that indicates the source serial channel for the LLP packet. |
+  | Data Type | The type of data contained in the LLP packet (i.e. Frame Start, Frame End, RAW types, YUV types, RGB types, etc.). |
+  | Virtual Channel | A number representing a virtual channel identifier. Virtual channel identifiers designate separate logical channels for data flows interleaved in the data path. |
+  | ECC | The Error Correction Code (ECC) detects bit errors in the header. |
+  | Word Count | The number of bytes in a long packet (i.e. payload size or bytes per line) and does not include the header or footer bytes. |
+  | Line Count | The Line Count value comes from the Word Count field in Line Start and Line End packets, which are optional. |
+  | Frame Count | The Frame Count value comes from the Word Count field in Frame Start and Frame End packets, which are mandatory, but there are no requirements on what the value is set to. |
```

Comment:
I'm kinda on the fence about including 'which are mandatory'. This help file doesn't seem like the appropriate place to teach CSI-2 protocol fundamentals, and mostly expect people reviewing this either know it already or should be learning it somewhere else.

Reply:
I agree.  I removed the "which are mandatory" from the description, but I kept the section about there being no requirement on what the value is set to.  It can be confusing if you see Frame Start and Frame End packets, but the Frame Count does not change.  However, the part about Frame Start/Stop packets being mandatory and Line Start/Stop packets being optional does seem beyond the scope of this document.